crates/services/p2p/src/peer_manager.rs

```diff
@@ -86,8 +83,17 @@ impl PeerManager {
         }
     }
 
-    pub fn is_reserved_peer(&self, peer_id: &PeerId) -> bool {
-        self.reserved_peers.contains(peer_id)
+    pub fn handle_gossip_score_update<T: Punisher>(
```

Comment:
maybe consider changing the function name to match the pattern of `update_app_score`

Reply:
app score gets updated by PeerManager, but gossipsub does not, this one just handles the post-update consequences